From 06871e666d474bdb14c7526e367cbaffbb83609c Mon Sep 17 00:00:00 2001
From: Wolfgang Schweer <wschweer@arcor.de>
Date: Sun, 3 Feb 2019 12:59:17 +0100
Subject: [PATCH] Set adjusted cd_type for incomplete BD image

If COMPLETE=0 is set, the content should rather be "bluray/not_complete" instead
of just "not_complete". This would allow one to build offline usable Edu BDs.
---
 tools/start_new_disc | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/tools/start_new_disc b/tools/start_new_disc
index 21f412f..f2abcd8 100755
--- a/tools/start_new_disc
+++ b/tools/start_new_disc
@@ -199,7 +199,11 @@ if [ $DISKNUM = 1 ] ; then
             fi
         fi
     else
-        echo "not_complete" > $CDDIR/.disk/cd_type
+        if [ $DISKTYPE = BD ] ; then
+            echo "bluray/not_complete" > $CDDIR/.disk/cd_type
+        else
+            echo "not_complete" > $CDDIR/.disk/cd_type
+        fi
     fi
 
     if [ $NUM_ARCHES -gt 1 ]; then
-- 
2.20.1

